spec/controllers/quotes_controller_spec.rb

```diff
@@ -1,66 +1,178 @@
 require 'rails_helper'
 
 RSpec.describe QuotesController, type: :controller do
-  let(:quote) { create(:quote) }
+  let(:user) { create(:user) }
```

Comment:
To avoid confusion with having `user` and `user_1` feel free to use things like:

`let(:molly) { FactoryGirl.create :user }`

I'm not sure if I can speak for everyone, but I prefer to write the full `FactoryGirl.create` so that it's explicit as to what we're using.